javaparser-core/src/main/java/com/github/javaparser/ast/Modifier.java

```diff
@@ -37,7 +37,8 @@
     VOLATILE,
     SYNCHRONIZED,
     NATIVE,
-    STRICTFP;
+    STRICTFP,
+    TRANSITIVE;

```

Comment:
I put it there because JP matches all modifiers in every location where modifiers are allowed. The idea is to check if the correct modifiers are used by validating the AST, which is not implemented yet: #594 . Or did you mean something else?